lib/minify.js

```diff
@@ -148,20 +150,32 @@ async function minify(files, options) {
     if (files instanceof AST_Toplevel) {
         toplevel = files;
     } else {
-        if (typeof files == "string") {
+        if (typeof files == "string" || (options.parse.spidermonkey && !Array.isArray(files))) {
```

Comment:
This can be just `if (!Array.isArray(files))`

Reply:
unfortunately not, it breaks 3 tests that pass an object 😬

Reply:
Oh, I forgot about passing filename + source as an object :)